### DISCUSSION TITLE
数据段结构混淆

### DISCUSSION THREAD

**Discussion by Soneoylys** [enfein/mieru#107] [Ideas]
基于TOTP的数据段结构混淆:
是否可以实现让客户端和服务端使用同一个TOTP的SEED, 在两边同步时间后利用生成的PSK来决定数据段的排放方式,
并且让服务端以可定义/随机的倍率发送无意义的数据包, 客户端直接丢弃不符合当前PSK数据段定义格式的包, 从而降低包一致性.

**Comment by enfein** [comment]
数据段结构需要混淆的主要原因是填充字段的熵比较低。除了上面所说的，还有一些更简单的办法，例如根据双方商定的映射重新排布每一个字节而不是每一个数据段的位置。根据实际需要，在未来我们可能会使用某些混淆策略。
